usr/src/test/smartos-test/README

```diff
@@ -101,3 +111,65 @@ It is important when adding or removing tests that the IPS manifests are
 updated. If new test packages are created, they should be added to
 $SRC/Makefile.testarchive.
 
+
+How We Use This:
+
+Every release we pull the release ISO and the matching test .tgz file down.
+For example:
+
+kebe(~/ws/ij-cr)[141]% mls -lt /Joyent_Dev/public/SmartOS/20230323T000605Z | egrep "iso|tests-"
+-rwxr-xr-x 1 Joyent_Dev     606420992 Mar 22 22:43 smartos-20230323T000605Z.iso
+-rwxr-xr-x 1 Joyent_Dev      12946513 Mar 22 22:43 tests-release-20230323-20230323T000605Z.tgz
+kebe(~/ws/ij-cr)[0]% 
+
+We then use a test machine (currently a VMware VM, which does cause some
+inconsistencies in the BHYVE/VMM results) which has:
+
+	16GB of RAM (though 8 would probably suffice)
+
+	A 100GB disk for `zones` (a smaller disk can cause ENOSPC problems
+	sometimes)
+
+	Three 30GB disks for setting up in DISKS for ZFS testing.  (fewer
+	can cause ZFS test failures)
+
+	An NIC that can reach the public Internet.
+
+We then perform a fresh installation.  The installation needs to have a bit
+of manual intervention.
+
+	The pool should not include the three 30GB disks. We do this by
+	selecting "manual" at pool creation time, and create the pool with
+		"zpool create -f -B zones c1t0d0"
+
+We also select "zones" as the bootable pool and we say yes to pkgsrc in the
+global zone.
+
+Once installed, we detach the ISO and boot from the `zones` pool.  After
+that, we pull the tests tarball and put it into /tmp/test/.  Inside /tmp/test
+we run:
+
+	gtar -xzf tests-release-20230323-20230323T000605Z.tgz
+		(Extracts tests)
+
+	touch /lib/sdc/.sdc-test-no-production-data
+		(let smartos-test know we're okay with this)
+
+	export DISKS="c1t2d0 c1t3d0 c1t4d0"
+		(your disk names may vary)
+
+	./opt/smartos-build/bin/smartos-build -c ./tests-release-20230323-20230323T000605Z.tgz
+		(Configure for testing)
+
+	/opt/smartos-build/bin/smartos-build -e ./tests-release-20230323-20230323T000605Z.tgz
+		(Run the tests!
+		 You can also uses ./opt/ as /opt is copied from here)
```

Comment:
Nit: just noticed the typo here`s/uses/use/`

Reply:
will fix.